static/stable/stage/main.yml

```diff
@@ -84,6 +84,13 @@ edge:
       - v1
     isBeta: true
 
+idmsvc:
+  title: "Directory and Domain Services"
+  api:
+    versions:
+      - v1
+    isBeta: true
```

Comment:
Copied from `beta/stage` directory; Would this be `false`?

Reply:
Nope, beta means the app is still in development. You can keep it.